charts/nova/templates/deployment-novncproxy.yaml

```diff
@@ -144,9 +145,14 @@ spec:
               readOnly: true
             - name: pod-shared
               mountPath: /tmp/pod-shared
+            {{- if $vencrypt_enabled }}
+            - name: {{ .Values.secrets.tls.compute_novnc_proxy.vencrypt.internal }}
+              mountPath: /etc/pki/nova-novncproxy
+              readOnly: true
+            {{- end }}
+
```

Comment:
This seems to have changed, I don't think this works for us or does it?

Reply:
i think it will work and be better because it doesn't use subpath